src/scripts/git_data.py

```diff
@@ -13,6 +17,36 @@ def get_last_commit_date(file_path):
         print(f"An error occurred: {e}")
     return None
 
+def get_last_commit_date(file_path):
+    try:
+        # get the last commit date as "September 12, 2022"
+        GITHUB_TOKEN = os.environ['GITHUB_TOKEN']
+
+        if GITHUB_TOKEN is None:
+            print('Github token not found')
+            return get_last_commit_date_by_terminal_log(file_path)
```

Comment:
we can get rid of that function and return None or "N/A" here.

Reply:
ae35db4731c468ac1222cdb723d49afb174fad08